DistroLauncher/installer_policy.h

```diff
@@ -147,15 +147,12 @@ namespace Oobe
                 return exitCode;
             }
 
-            GetExitCodeProcess(process, &exitCode);
-            // The Flutter installer exits 0 on some crashes.
-            // And sometimes reports crash even though Subiquity has finished. This gets fixed in [UDI PR
-            // 786](https://github.com/canonical/ubuntu-desktop-installer/pull/786)
-            if (exitCode == 0) {
-                g_wslApi.WslLaunchInteractive(L"grep -E 'EXITED|DONE' /run/subiquity/server-state", FALSE, &exitCode);
-                DWORD clearExitCode;
-                g_wslApi.WslLaunchInteractive(L"clear", FALSE, &clearExitCode);
```

Comment:
You didn’t explain why you removed this clear execution, will we get on the console some debug output with grep?

As this only impact people with latest dev Windows version, I think we can clear that out before requalifying a new image.

Reply:
That was a mistake indeed, as I was experimenting with the code, I removed more than needed. 

This clear command is bugging me a little bit because it unconditionally clears the user's screen without considering whether it was doing something before on the console that should not be disturbed. Ideally, that clearing should only happen for the text-mode installation. But that's something I'll still clear my mind on to find the better way to do and then it will be a topic for another PR, not this one.